src/Schema/Blueprint.php

```
@@ -501,7 +501,8 @@ protected function createIndexName($type, array $columns)
             $table = $this->connection->getTablePrefix() . $table;
         }
 
-        $index = strtolower($table . '_' . implode('_', $columns) . '_' . $type);
+        $columnNames = array_is_list($columns) ? $columns : array_keys($columns);
```

PHPStan (2.2.8)

[error] Call to function array_is_list() with list<string> will always evaluate to true.
Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your %configurationFile%.

(function.alreadyNarrowedType)